src/test/java/org/apache/aurora/scheduler/mesos/MesosTaskFactoryImplTest.java

```diff
@@ -23,16 +24,11 @@
 
 import org.apache.aurora.GuavaUtils;
 import org.apache.aurora.common.testing.easymock.EasyMockTest;
-import org.apache.aurora.gen.AppcImage;
-import org.apache.aurora.gen.AssignedTask;
-import org.apache.aurora.gen.Container;
-import org.apache.aurora.gen.DockerContainer;
-import org.apache.aurora.gen.DockerImage;
-import org.apache.aurora.gen.DockerParameter;
-import org.apache.aurora.gen.Image;
-import org.apache.aurora.gen.MesosContainer;
-import org.apache.aurora.gen.ServerInfo;
-import org.apache.aurora.gen.TaskConfig;
+
+import org.apache.aurora.gen.*;
```

Comment:
nit: maybe we can remove the import with a wildcard, so we can easily see which imports are added by us

Reply:
Done